tools/FakeItEasy.Build/Program.cs

```diff
@@ -83,36 +84,43 @@ public static void Main(string[] args)
             Target(
                 "unit",
                 DependsOn("build", "testsDirectory"),
-                () => RunTests("unit"));
+                forEach: TestSuites["unit"],
+                action: testDirectory => RunTests(testDirectory));
 
             Target(
                 "integ",
                 DependsOn("build", "testsDirectory"),
-                () => RunTests("integ"));
+                forEach: TestSuites["integ"],
+                action: testDirectory => RunTests(testDirectory));
 
             Target(
                 "spec",
                 DependsOn("build", "testsDirectory"),
-                () => RunTests("spec"));
+                forEach: TestSuites["spec"],
+                action: testDirectory => RunTests(testDirectory));
 
             Target(
                 "approve",
                 DependsOn("build", "testsDirectory"),
-                () => RunTests("approve"));
+                forEach: TestSuites["approve"],
+                action: testDirectory => RunTests(testDirectory));
+
+            Target("get-version", () => version = Read(ToolPaths.GitVersion, "/showvariable NuGetVersionV2"));
+
+            Target(
+                "pack-projects",
+                DependsOn("build", "outputDirectory", "pdbgit", "get-version"),
+                forEach: ProjectsToPack,
+                action: project => Run("dotnet", $"pack {project} --configuration Release --no-build --output {OutputDirectory} /p:Version={version}"));
+
+            Target(
+                "pack-nuspecs",
+                DependsOn("build", "outputDirectory", "pdbgit", "get-version"),

```

Comment:
This one doesn't actually depend on `pdbgit`. It just creates an empty package with dependencies on other packages. In fact, it doesn't even depend on `build` ;)

Reply:
Ah, ok. I'l remove the deps on `pdbgit` and `build`.